server/src/services/album.service.ts

```diff
@@ -212,14 +215,14 @@ export class AlbumService {
   async addUsers(auth: AuthDto, id: string, dto: AddUsersDto): Promise<AlbumResponseDto> {
     await this.access.requirePermission(auth, Permission.ALBUM_SHARE, id);
 
-    const album = await this.findOrFail(id, { withAssets: false });
+    const album = await this.findOrFail(id, { withAssets: true });
```

Comment:
The old `return` query of this function returned assetCount, so I adapted that to happen here now instead. The mapping still happens without assets in both cases.

Reply:
Yeah we can't be using withAssets: true here. For large albums this loads way too much data from the database.